app/views/layouts/_logo.html.erb

```diff
@@ -1,11 +1,21 @@
 <a href="<%= root_path %>" class="site-logo" aria-label="<%= t("views.main.aria_home", community: community_name) %>">
+  <!-- its a little explicit for the moment but it will make
```

Comment:
Like I mention, the if else here is explicit, but I prefer it to be explicit since it will make removing the feature flag much easier next week.

Reply:
This also changes the way things are rendered a bit and hence some tests and dynamic logo updates needed to be updated alongside.